src/main/java/uk/gov/hmcts/reform/preapi/config/SecurityConfig.java

```diff
@@ -32,7 +32,9 @@ public class SecurityConfig {
         "/users/by-email/**",
         "/reports/**",
         "/audit/**",
-        "/error"
+        "/error",
+        "/invites",
```

Comment:
Why do we need this one?

Reply:
yeh, I need to check if the user is new by calling GET /invites?email=someone@somewhere.com and if i get a response I'll POST /invites/redeem/someone@somewhere.com

Would that need to be /invites/redeem/** in the SecurityConfig actually or do params not count?

Reply:
Just tested to make sure- it does ignore the params